pkg/frontend/session.go

```diff
@@ -47,15 +47,18 @@ const (
 )
 
 type TxnHandler struct {
-	storage engine.Engine
```

Comment:
just keep it 'storage'. In the frontend's view, the concept storage includes everything involved with the storage and the transaction.

Reply:
'Storage' has been splited to txn client and engine

Reply:
> 'Storage' has been splited to txn client and engine

Just keep the concept 'Storage'

Reply:
ok